Tools/XMLtoMD/Directory.Packages.props

```diff
@@ -0,0 +1,14 @@
+<Project>
```

Comment:
Any reason for not just having a single Directory.Packages.props at the root of repo?
Nothing wrong with this, but it seems like splitting the thing that is designed to centralize.